crates/astria-core/src/crypto.rs

```diff
@@ -0,0 +1,83 @@
+use std::fmt::{
+    self,
+    Debug,
+    Formatter,
+};
+
+use ed25519_consensus::{
+    Signature,
+    SigningKey as Ed25519SigningKey,
+    VerificationKey,
+    // VerificationKeyBytes,
```

Comment:
can remove

Reply:
Done in [6b05e11](https://github.com/astriaorg/astria/pull/1074/commits/6b05e114d3a3860868252108f2fc7be22636e433).